docs/source/development_tips.rst

```diff
@@ -143,14 +150,14 @@ In some cases you may not want your check to run on a CRON type schedule but sti
 * The explicit method is to use the ``manual_checks`` schedule set up in exactly the same way as any of the other schedules.  This schedule is set to run effectively never by setting the CRON in the app to Feb 31st.
 * The other method is to leave the schedule empty but include the ``display`` property and list the environments that you wish the check to appear. eg.
 
-.. code-block::
+.. code-block:: JSON
 
    {
-       "my_first_check": {
-           "title": "My first check",
-           "group": "Awesome test checks",
-           "schedule": {},
-           "display": ["data", "staging"]
-       }
+      "my_first_check": {
```

Comment:
Indentation looked fine in the previous version.

Reply:
fixed